spec/requests/requests_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'managing topics', :type => :request do
```

Comment:
I generally name my request specs after the controller -  Topic requests - One set of request specs per controller.
Also, name your request spec file that way (e.g. _topic_requests_spec.rb_)